config/shared_config.go

```diff
@@ -193,6 +195,19 @@ type SharedConfig struct {
 	//
 	// use_fips_endpoint=true
 	UseFIPSEndpoint aws.FIPSEndpointState
+
+	// Specifies which defaults mode should be used by services.
+	//
+	// defaults_mode=auto
+	DefaultsMode aws.DefaultsMode
```

Comment:
nit should this documentation refer to `standard` instead of `auto` since auto is experimental?